apps/i18n/common/en_us.json

```diff
@@ -632,6 +632,7 @@
   "dropletBlock_playSound_description": "Plays the MP3 sound file from the specified URL.",
   "dropletBlock_playSound_param0_description": "The URL to a sound file. Can be a project asset name or external URL.",
   "dropletBlock_playSound_param1_description": "If true, loop the sound until asked to stop.",
+  "dropletBlock_playSpeech_description": "Plays the specified text as speech in the specified language and gender. Powered by Microsoft Azure.",
```

Comment:
nit: if removing language, we should also remove `in the specified language` here